samples/sensor/sensor_shell/x_nucleo_iks01a3.overlay

```diff
@@ -4,6 +4,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/ {
+	virtual-sensors {
```

Comment:
The `virtual-sensors` concepts can be leveraged by the proposed `senss` (#45370) for virtual sensor device tree node definitions.    

But from `senss`'s view, there're some more things need to consider:
- `senss` can only enumerate and management `senss sensors`(implemented the `senss sensor` API)
    So, suggest define a  device tree parent node for all `senss sensors`, such as `sensor-subsystem`, then `senss` can enumerate all `senss sensors` from `sensor-subsystem` node. 
    This can also help isolate `senss sensors` nodes with zephyr sensor device nodes for existing API or V2 API (#44098), and both of them can inherit the base sensor properties, such as vendor, model, friendly-name.   

- `reporters` phandle property for `senss virtual sensor` must also point to `senss sensor(s)` (either virtual or physical) 

- `senss physical sensor` is not a zephyr physical device sensor (implement the existing API, or V2 API).  

    Unlike `senss virtual sensor` takes other `senss sensor(s)` (virtual or physical) as input reporters, a `senss physical sensor` has no `senss sensors` as inputs (`reporters` not defined or empty), it's just a data producer, collect underlying physical device's data etc. 
    The underlying device can be zephyr physical sensor device (existing API or V2), or even can be any other devices, such as a wifi/ble device, or even pure sw which an produce data and can be abstract as a sensor. This can make `senss` more extensible and powerful.

    So, in #45370 , we defined the `underlying-device` for a `senss physical sensor` to reference a none `senss sensor` device. 

    For example: 

    ```dts
    /{
        &senss {
                hinge_angle : hinge-angle {
                       compatible = "senss,hinge-angle"
                       reporters =<&base_accel &lid_accel>
                }
                base_accel: base-accel {
                       compatible = "senss,phy-accel"
                       underlying-device = <&bmi_spi>
                }
                lid_accel: lid-accel {
                       compatible = "senss,phy-accel"
                       underlying-device = <&bmi_i2c>
                }
        }

        &spi0 {
               bmi_spi: bmi@3 {
                     compatible = "bosch,bmi160"
               }
        }

        &i2c0 {
               bmi_i2c: bmi@68 {
                      compatible = "bosch,bmi160"
               }
        }
    }
    ```
    `bmi_i2c` and `bmi_spi` are non `senss sensors` nodes.